module.txt

```diff
@@ -16,6 +16,18 @@
         {
             "id": "Signalling",
             "minVersion": "0.1.1"
+        },
+	{
+            "id": "SegmentedPaths",
```

Comment:
why do you need these modules. SegmentedPaths is a dependency of Rails. you are not using ModularComputers or MarcinScIncubator.

Reply:
I added those while trying to fix an issue where it couldn't find some classes from those modules and forgot to remove them when I found the actual issue. I'll get rid of them right now.